components/hal/esp32c2/include/hal/clk_tree_ll.h

```diff
@@ -22,6 +22,7 @@
 extern "C" {
 #endif
 
+#undef MHZ
 #define MHZ                 (1000000)
```

Comment:
I think, we have MHZ() macros in Zephyh somewhere, better to use them and not introduce new ones.

Reply:
Similar situation here. The conflict is between standard Zephyr macro and IDF macro.
Changing all IDF macros will affect all SoC's, requiring switching all IDF macros to Zephyr's.

Reply:
We agreed this change should go in a separate PR, as it affects other SoC's as well.